app/web/src/features/communities/discussions/DiscussionPage.tsx

```diff
@@ -88,55 +90,64 @@ export default function DiscussionPage() {
         <CircularProgress />
       ) : (
         discussion && (
-          <CommunityBase
-            communityId={discussion.ownerCommunityId}
-            defaultTab="discussions"
-          >
-            {() => (
-              <div className={classes.root}>
-                <div className={classes.header}>
-                  <HeaderButton
-                    onClick={() => history.goBack()}
-                    aria-label={PREVIOUS_PAGE}
+          <CommunityBase communityId={discussion.ownerCommunityId}>
+            {({ community }) => (
+              <>
+                {community.mainPage && (
+                  <PageHeader
+                    page={community.mainPage}
+                    className={classes.header}
+                  />
+                )}
+                <CommunityPageSubHeader community={community} />
```

Comment:
```suggestion
                <CommunityPageSubHeader community={community} defaultTab="discussions" />
```
I think that's why what I mentioned before is not updating to the correct tab when navigating to individual discussions

Reply:
also noticed that the edit community page shows the overview tab as selected while that should remain 'local info' 
I decided not to fix that yet, not to broaden the scope of my commits.
Ill have a quick look now though :)

Reply:
yeah that seems to do it for both discussions page and for edit community page